test/Parse/ConditionalCompilation/if_darwin.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %swift -typecheck %s -verify -target arm64-apple-ios8.0
```

Comment:
Sorry for the confusion here, I edited my last comment because I realized `%target-typecheck-verify-swift` isn't quite right either as it assumes the target of the active test configuration, but you want to specify it yourself. So `%swift -typecheck %s -verify` is indeed correct.

With this testing configuration you should actually be able to remove `REQUIRES: OS=ios`, though you will need to add `-parse-stdlib` to the invocation. I now realize that it's necessary in this case because the CI might not have built the stdlib for the given target.

This also means you could add additional `RUN` lines for all the other platforms you want `Darwin` to alias to, which will make sure it matches all of them.

Reply:
Should now be fixed by https://github.com/apple/swift/pull/59426/commits/3efb7a816d7c09585f0a143405254cebdcb7368b

Reply:
Thanks!